tools/site_admin/manage_site_access_privileges.php

```diff
@@ -45,7 +45,7 @@
     'pp_limit_value' => _("Post-processing projects limit, -1 means unlimited"),
 ];
 
-$username = array_get($_POST, 'username', array_get($_GET, 'username', null));
+$username = array_get($_POST, 'username', array_get($_GET, 'username', ''));
```

Comment:
I changed the fallback from `null`  (which made the type `?string`) to the empty string. This seemed slightly better as it avoids having to think about handling `null` downstream.

Reply:
I ended up rescinding this change as it is common for `$username` to be `null` and it is better to match the current expectations rather than break from it.